tests/Microsoft.Fx.Portability.MetadataReader.Tests/ReflectionMetadataInfoTests.cs

```diff
@@ -46,7 +46,7 @@ public void UnresolvedAssemblyTest()
             Assert.Equal(s_expectedResult.Count(), actual.Count());
 
             // Use this instead of Assert.Equal so it will output the missing item
-            foreach (var items in s_expectedResult.Zip(actual, Tuple.Create))
+            foreach (var items in actual.Zip(s_expectedResult, Tuple.Create))
```

Comment:
That's what I had thought, but it was failing here with some of the changes and I noticed the 'expected' and 'actual' were switched. Caused a bit of confusion, so I switched the parameters for next time it fails.